### ISSUE TITLE
Utilize separate language server extension en lieu of internal support

### ISSUE THREAD

**Issue by hansec** [krvajal/vscode-fortran-support#105]
Hey, I am developing the [Fortran Language Server](https://github.com/hansec/fortran-language-server) and a companion [extension for VSCode](https://marketplace.visualstudio.com/items?itemName=hansec.fortran-ls). I just updated the VSCode extension (v0.2.5) to work properly with this extension.

As there are settings that are unique to the language server I think it is best to keep this functionality in a separate extension at least for now. Once things stabilize with the language server interface it might make sense to merge the language server support into a single extension. For now though, I wanted to suggest you remove that option from your extension and point people to the language server extension. That way it is clear what settings effect which functionality and where people should report bugs.

I also wanted to say thanks! I am so glad there is actively maintained Fortran support for VSCode. So many of the Fortran editor packages I have dealt with in the past get abandoned without updates for new syntax and bug fixes.

**Comment by starship863** [comment]
As a user, I must say thanks for all the authors for these two awesome extensions. They make VSCode a really cool IDE for fortran developers.

**Comment by pedro-ricardo** [comment]
Done in #120

**Comment by JHenneberg** [comment]
Hey. I am using Modern Fortran and the fortran-language-server on a windows machine and it is working, althoug in the settings and in a pop-up warning it is written it is not availuable for windows. Is this warning outdated?

**Comment by krvajal** [comment]
It is not available for windows because I was not able to verify that it works. 
Interesting though to know your setup. We could update the warning and say that it could work but it is not supported. Not sure if that will make more sense.

**Comment by JHenneberg** [comment]
Ah ok. I understand. My Setup is:
WIN 10
VS Code 1.32.3
Modern Fortran 2.0.2
Fortran IntelliSense 0.4.0
fortran-language-server 1.7.3

```JSON
    "fortran.linterEnabled": false,
    "fortran.preferredCase": "uppercase",
    "fortran.useLanguageServer": true,
    "fortran-ls.variableHover": true,
    "fortran-ls.enableCodeActions": true,
    "fortran-ls.notifyInit": true
```

**Comment by starship863** [comment]
> Ah ok. I understand. My Setup is:
> WIN 10
> VS Code 1.32.3
> Modern Fortran 2.0.2
> Fortran IntelliSense 0.4.0
> fortran-language-server 1.7.2
> 
> ```json
>     "fortran.linterEnabled": false,
>     "fortran.preferredCase": "uppercase",
>     "fortran.useLanguageServer": true,
>     "fortran-ls.variableHover": true,
>     "fortran-ls.enableCodeActions": true,
>     "fortran-ls.notifyInit": true
> ```

I can also confirm that it is working on my win10 laptop nicely with similar setup.

**Comment by hansec** [comment]
Yep, the language server should work just fine on Windows. I think these errors should go away when the above PR (#120) is released. In the meantime @JHenneberg @starship863 and others could disable "Use Language Server" in the "Modern Fortran" settings, since the "Fortran IntelliSense" extension is providing the language server interface.

Hopefully, it won't be too long before the features/interface of the language server stabilize and we can reintegrate the language server into this extension and remove the separate "IntelliSense" extension.

**Comment by krvajal** [comment]
Fixed in [v2.1.0](https://github.com/krvajal/vscode-fortran-support/releases/tag/2.1.0)
